tests/helpers/editor-helpers.js

```diff
@@ -29,6 +29,29 @@ export function titleRendered() {
     });
 }
 
+// replaces the title text content with HTML and returns once the HTML has been placed.
+// takes into account converting to plaintext.
+export function replaceTitleHTML(HTML) {
```

Comment:
Done, I'm working on a PR which fixes the issues that testing has with some newly introduced side-effects and then to re-enable the skipped tests and expand the test coverage.

I'll include these lessons / make a more generalised helper in that.